tests/hardhat/Comptroller/accountLiquidityTest.ts

```diff
@@ -314,53 +314,6 @@ describe("Comptroller", () => {
         .to.be.revertedWithCustomError(comptroller, "MarketAlreadyListed")
         .withArgs(vToken.address);
     });
-
-    it("reverts if not 'in' any markets", async () => {
```

Comment:
we do not need this test anymore? Can you elaborate on that?

Reply:
https://github.com/VenusProtocol/isolated-pools/pull/167/commits/2e46c045df3cb3b804448b4045277351cf016a17

Reply:
Thanks got it